src/Sarif/Errors.cs

```diff
@@ -26,6 +26,7 @@ public static class Errors
         private const string ERR997_ExceptionLoadingAnalysisTarget = "ERR997.ExceptionLoadingAnalysisTarget";
         private const string ERR997_ExceptionInstantiatingSkimmers = "ERR997.ExceptionInstantiatingSkimmers";
         private const string ERR997_OutputFileAlreadyExists = "ERR997.OutputFileAlreadyExists";
+        internal const string ERR998_IncompatibleRuleDetected = "ERR997.IncompatibleRuleDetected";
```

Comment:
>ERR998_IncompatibleRuleDetected [](http://example.com/codeflow?start=30&length=31)

ERR998_IncompatibleRulesDetected

Note the plural form. It takes two rules to be incompatible. :)